src/annotator/test/guest-test.js

```diff
@@ -487,6 +492,22 @@ describe('Guest', () => {
         assert.notCalled(guest.crossframe.call);
       }
     });
+
+    it('reposition the adder on window "resize" event', () => {
+      sandbox.stub(guest, '_repositionAdder').callThrough();
+      sandbox.stub(guest, '_onSelection');
+      window.dispatchEvent(new Event('resize'));
+
+      // If the adder is not displayed on 'window.resize' event does nothing
+      assert.called(guest._repositionAdder);
+      assert.notCalled(guest._onSelection);
+
+      // If the adder is displayed then it calls 'guest._onSelection'
```

Comment:
Maybe reword this a little...

"Calls _onSelection() if the adder is visible"

And I would actually just make this a second test since its testing a different code path.

Reply:
I have split the tests in two. I name the test more in terms of what it does, more than what function calls.